generate_workspace/src/main/java/com/google/devtools/build/workspace/maven/Rule.java

```diff
@@ -16,18 +16,18 @@
 
 import com.google.common.base.Preconditions;
 import com.google.common.collect.Sets;
-
-import java.lang.invoke.MethodHandles;
-import java.util.logging.Logger;
-import javax.annotation.Nullable;
 import org.eclipse.aether.artifact.Artifact;
 
+import javax.annotation.Nullable;
+import java.lang.invoke.MethodHandles;
 import java.util.Objects;
 import java.util.Set;
+import java.util.logging.Logger;
 
 /**
  * A struct representing the fields of maven_jar to be written to the WORKSPACE file.
  */
+//TODO(petros): Kill this after refactoring resolvers.
```

Comment:
It does effectively the same thing, except using aether classes. The original class uses maven versions of classes like Artifact and what not. 

Because of this, I found that augmenting `Rule` class required a considerable amount of refactoring across the codebase. 

Since I am going to rip the resolvers out, I thought it would be a better use of time to keep both classes.